notebooks/custom_kinfraglib/1_3_custom_filters_synthesizability.ipynb

```diff
@@ -0,0 +1,2349 @@
+{
```

Comment:
<ul><li>Check the Enamine statement above and maybe reformulate: not the building blocks can be synthesized on demand, but the respective compounds from these building blocks ...</li><li>Data Warrior ... can be used <strong>for</strong> searching building ...</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sonjaleo/KinFragLib/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>